helsingborg-stad/helsingborg-io-sls-api
# Fix/move queries folder

## Explain the changes you’ve made
Move queries folder to viva-ms service 

## Explain why these changes are made
the query handlers were importing types from a path that did not exist, hence the building were failing

## Explain your solution
Moved the query folder into viva-ms service folder. Changed the path to CaseItem path and also the path to query folder in setcasecompletions lambda

## How to test

Concrete example:

1. Checkout this branch
2. run `sls deploy` within service folder for `viva-ms`
3. tests and deployment should be successful

## Tested environments

- [] Your local Serverless environment.
